src/app/researcher/study/request/[memberSlug]/actions.ts

```diff
@@ -20,6 +21,18 @@ export const onCreateStudyAction = researcherAction(async ({ memberSlug, studyIn
 
     const member = await getMemberFromSlugAction(memberSlug)
 
+    // Verify researcher is allowed to submit studies to this organization
+    const memberUser = await db
+        .selectFrom('memberUser')
+        .select('id')
+        .where('userId', '=', userId)
+        .where('memberId', '=', member.id)
+        .executeTakeFirst()
```

Comment:
nitpicking here but we can extract this query out (as an identical query is in the `page.tsx` file) into `queries.ts` but not a showstopper